tests/test_core/test_new_alignment.py

```diff
@@ -3848,6 +3876,41 @@ def test_get_feature():
     assert feat.get_slice().to_dict() == dict(x="AAA", y="CCT")
 
 
+def test_alignment_sample_with_replacement():
+    # test with replacement -- just verify that it rnus
+    alignment = new_alignment.make_aligned_seqs(
+        {"seq1": "GATC", "seq2": "GATC"}, moltype="dna"
+    )
+    sample = alignment.sample(n=100, with_replacement=True)
+    assert len(sample) == 100
+    # ensure that sampling with replacement works on single col alignment
+    alignment1 = new_alignment.make_aligned_seqs(
+        {"seq1": "A", "seq2": "A"}, moltype="dna"
+    )
+    result = alignment1.sample(with_replacement=True)
+    assert len(result) == 1
+
+
+def test_alignment_sample_tuples():
+    ##### test with motif size != 1 #####
+    alignment = new_alignment.make_aligned_seqs(
+        {
+            "seq1": "AACCDDEEFFGGHHIIKKLLMMNNPP",
+            "seq2": "AACCDDEEFFGGHHIIKKLLMMNNPP",
+        },
+        moltype="protein",
+    )
+    # ensure length correct
+    sample = alignment.sample(n=10, motif_length=2)
+    assert len(sample), 20
+    # test columns alignment preserved
+    seqs = list(sample.to_dict().values())
+    assert seqs[0], seqs[1]
```

Comment:
should that be `seqs[0] == seqs[1]`?

Reply:
good catch! issue from copy pasta from unittesting